presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1000,6 +1104,8 @@ Examples:
 
     CALL iceberg.system.expire_snapshots(schema => 'schema_name', table_name => 'table_name', snapshot_ids => ARRAY[10001, 10002]);
 
+=======
+>>>>>>> Move the description of Iceberg procedures to a dedicated subheading
```

Comment:
Here exists a conflict. Please handle it first.

Reply:
conflicts fixed.